pkg/config/engine/containerd/config_v2.go

```diff
@@ -22,6 +22,17 @@ import (
 	"github.com/NVIDIA/nvidia-container-toolkit/pkg/config/toml"
 )
 
+type ctrdCfgV2Runtime struct {
```

Comment:
Can we add the below to ensure this new type adheres to the engine.RuntimeConfig interface?
```
var _ engine.RuntimeConfig = (*ctrdCfgV2Runtime)(nil)
```